### PR TITLE
Fixes Issue #68.

### PR DESCRIPTION
**Pull request by at88mph**
Add LDAP login support, consistent with the other login functions.

### CODE REVIEW

File: example/auth_ldap.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// file: example/auth_userpass.js
```
Comment (kr1sp1n) [inline example/auth_ldap.js]: Just change the line to `example/auth_ldap.js` 😉

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-vault",
-  "version": "0.6.1",
+  "version": "0.6.2",
```
Comment (kr1sp1n) [inline package.json]: Please do not bump version by yourself. This is handled by me 😉

**Comment by at88mph** [comment]
Thank you, @kr1sp1n.  Post review changes committed and pushed.

**Comment by at88mph** [comment]
Are the integration failures a known issue?

**Comment by kr1sp1n** [comment]
@at88mph Please merge latest `master` branch where I fixed the integration test on Travis CI 🙏

**Comment by at88mph** [comment]
Merged in master.  Travis tests pass.

**Comment by at88mph** [comment]
Is there an issue with `codecov`?  What is it waiting on?

**Comment by kr1sp1n** [comment]
@at88mph Don't know what's going on with codecov but I will merge now. Thanks for the PR 🙏

**Comment by at88mph** [comment]
Fantastic, thanks @kr1sp1n.  Is ti automagically published to NPM?

**Comment by kr1sp1n** [comment]
Not yet. Will take care of it asap.

**Comment by kr1sp1n** [comment]
@at88mph Just released your changes as v0.7.0
Thanks for your contribution! 🙏

**Comment by at88mph** [comment]
Sweet!  Thanks.

**Pull request merged**